src/bda/plone/cart/profiles/default/registry.xml

```diff
@@ -6,23 +6,24 @@
            interface='Products.CMFPlone.interfaces.IBundleRegistry'>
     <value key="enabled">True</value>
     <value key="compile">False</value>
-    <value key="jscompilation">++resource++cookie.functions.js</value>
+    <value key="jscompilation">++resource++bda.plone.cart/cookie.functions.js</value>
```

Comment:
out of scope of this pull request, but we should talk about switching from cookies to local storage for remembering the cart items